### ISSUE TITLE
yarn start internal error 

### ISSUE THREAD

**Issue by mohammadrezza** [calcom/docker#274]
I'm trying to run cal with docker and I get the following error:

```
+ yarn start
Internal Error: @calcom/skiff@workspace:packages/app-store/skiff: This package doesn't seem to be present in your lockfile; run "yarn install" to update the lockfile
    at P0.getCandidates (/calcom/.yarn/releases/yarn-3.4.1.cjs:435:5145)
    at yf.getCandidates (/calcom/.yarn/releases/yarn-3.4.1.cjs:391:1264)
    at /calcom/.yarn/releases/yarn-3.4.1.cjs:439:7693
    at of (/calcom/.yarn/releases/yarn-3.4.1.cjs:390:11070)
    at ge (/calcom/.yarn/releases/yarn-3.4.1.cjs:439:7673)
    at async Promise.allSettled (index 44)
    at async $s (/calcom/.yarn/releases/yarn-3.4.1.cjs:390:10398)
    at async /calcom/.yarn/releases/yarn-3.4.1.cjs:439:8205
    at async ti.startProgressPromise (/calcom/.yarn/releases/yarn-3.4.1.cjs:390:47579)
    at async je.resolveEverything (/calcom/.yarn/releases/yarn-3.4.1.cjs:439:6233)
```

**Comment by jaydrogers** [comment]
I just attempted to upgrade from `v3.0.10` → `v3.1.1` and experienced this same error on container initialization.

## Workaround: Downgrade to v.3.1.0
In your calcom/docker code (basically the code from this repo), you will see there is a `docker-compose.yml` file. That file has the image definition. **That file is version controlled, so don't touch it.**

What you can do is use [Docker Compose Overrides](https://docs.docker.com/compose/extends/) to override the image definition.

### What you need to do
1. Create a `docker-compose.override.yml` file in the root of the Git source folder
2. Paste in the contents below
3. Attempt to bring your containers up again

**docker-compose.override.yml:**
```yaml
# This file overrrides what is provided by Cal (or Calendoso)
version: '3.8'
services:
  calcom:
    image: calcom.docker.scarf.sh/calcom/cal.com:v3.1.0
```

🚨👉 This essentially sets the image to always be at `v3.1.0`. In the future, you will just need to update the file above to what version you want to use, but at least you'll explicit control over what version you want to run.

**Comment by krumware** [comment]
can you check again with :main?
:v3.1.2 will be available in about 20mins also for testing

**Comment by jaydrogers** [comment]
It's fixed with `v3.1.2` 👍

I saw 3.1.3 is out, but I was able to get it up and running with 3.1.2 without the error reported by @mohammadrezza

**Comment by krumware** [comment]
Thanks for the testing!
